### PR TITLE
[SW-2583] Expose "cross_validation_metrics_summary" on H2OMOJOModel

### PR DESCRIPTION
**Pull request by mn-mikke**

### CODE REVIEW

**Review by koniecsveta — APPROVED** [review]

File: py/tests/unit/with_runtime_sparkling/test_mojo.py [hunk 1]
```diff
@@ -195,3 +195,12 @@ def testMojoModelCouldBeSavedAndLoaded(gbmModel, prostateDataset):
     result = loadedModel.transform(prostateDataset).drop("detailed_prediction")
 
     unit_test_utils.assert_data_frames_are_identical(expected, result)
+
+
+def testGetCrossValidationSummaru():
+    mojo = H2OMOJOModel.createFromMojo(
```
Comment (koniecsveta) [inline py/tests/unit/with_runtime_sparkling/test_mojo.py:201]: nitpick: there is a typo in Summaru

Comment (mn-mikke) [reply py/tests/unit/with_runtime_sparkling/test_mojo.py:201]: Good catch! Thank you!

**Review by michalkurka — APPROVED** [review]
LGTM - but in further PRs I think we should clarify dichotomy of SW metric vs H2O metric

**Comment by mn-mikke** [comment]
>  I think we should clarify dichotomy of SW metric vs H2O metric

@michalkurka I've changed the functionality to show only SW metric names.

**Pull request merged**